### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by tmcw**
Add simple statistics, a statistical programming library. Website: http://simplestatistics.org/

### CODE REVIEW

File: files/simple-statistics/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Tom MacWright"
+github = "https://github.com/simple-statistics/simple-statistics"
+homepage = "http://simplestatistics.org/"
+description = "Statistical methods in readable JavaScript for browsers, servers, and people."
+mainfile = "simple-statistics.min.js"
```
Comment (jimaek) [inline files/simple-statistics/info.ini]: `simple_statistics.min.js`

File: files/simple-statistics/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "simple-statistics",
+  "repo": "simple-statistics/simple-statistics",
+  "files": {
+    "include": ["simple-statistics.js", "simple-statistics.min.js"],
```
Comment (jimaek) [inline files/simple-statistics/update.json]: The file names are not correct. The repo has them as `simple_statistics.min.js`
Also feel free to remove the unminified file, its not needed in production.

**Comment by tmcw** [comment]
:+1: thanks @jimaek, filenames are now updated

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**